core/src/main/java/com/oath/oak/Handle.java

```diff
@@ -85,7 +85,6 @@ boolean compute(Consumer<OakWBuffer> computer) {
     }
 
     ByteBuffer getSlicedByteBuffer() {
-        assert writeLock.isHeldByCurrentThread();
```

Comment:
Why to remove this? Is it under a read lock and read lock can not be checked? Please add a comment